docs/gke-installation.md

```diff
@@ -0,0 +1,126 @@
+
+    ```bash
+    export ADMIN_USER=user@email.com
+    gcloud auth login
+    ```
+
+3. Create a project or use an existing one
+
+    ```bash
+    export GKE_PROJECT=gke-clusters
+    gcloud projects create $GKE_PROJECT
+    ```
+
+## Creating the cluster
+
+You can use any method to create a GKE cluster (gcloud SDK, gcloud Console etc). The example
+given here is using the Google Cloud SDK.
+
+**Note:** Antrea is supported on Ubuntu hosts only for GKE cluster. Also, its must to select service
+CIDR at the time of cluster deployment.
+
+1. Create a GKE cluster
+
+    ```bash
+    export GKE_ZONE="us-west1"
+    export GKE_HOST="UBUNTU"
+    export GKE_SERVICE_CIDR="10.94.0.0/16"
+    gcloud container --project $GKE_PROJECT clusters create cluster1 --image-type $GKE_HOST \
+       --zone $GKE_ZONE --enable-ip-alias --services-ipv4-cidr $GKE_SERVICE_CIDR
+    ```
+
+2. Access your cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                      STATUS   ROLES    AGE     VERSION
+    gke-cluster1-default-pool-93d7da1c-61z4   Ready    <none>   3m11s   v1.14.10-gke.17
+    gke-cluster1-default-pool-93d7da1c-rkbm   Ready    <none>   3m9s    v1.14.10-gke.17
+    ```
+
+3. Create a cluster-admin ClusterRoleBinding
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user user@email.com
+    ```
+
+    **Note:** To create clusterRoleBinding, the user must have `container.clusterRoleBindings.create` permission.
+Use this command to enable it, if the previous command fails due to permission error. Only cluster Admin can
+assign this permission.
+
+    ```bash
+    gcloud projects add-iam-policy-binding $GKE_PROJECT --member user:user@email.com --role roles/container.admin
+    ```
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy antrea-node-init DaemonSet to enable kubelet to operate in CNI mode.
+
+    ```bash
+    kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.5.0/patches/gke/antrea-node-init.yml
+    ```
+
+2. Download and Update Antrea YAML
+
+    Download a released version of Antrea deployment YAML (>=v0.5.0). Update the `serviceCIDR` config in the
+antrea-gke.yml file with GKE_SERVICE_CIDR selected at the time of deploying GKE cluster.
+
+    ```bash
+    export TAG=v0.5.0
+    curl https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-gke.yml --output antrea-gke.yml
+
+    cat antrea-gke.yml
+    ...
+        # CIDR Range for services in cluster. It's required to support egress network policy, should
+        # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
+        serviceCIDR: 10.94.0.0/16
+    ...
+    ```
+
+3. Deploy Antrea
+
+    Deploy Antrea using `kubectl apply -f antrea-gke.yml`. It will deploy single replica of Antrea controller
+and Antrea agent to every worker nodes of the GKE cluster. After a successful deployment you should be able to see
```

Comment:
To be clear, maybe: and deploy Antrea agent to...

Reply:
nodes -> Nodes

Reply:
> To be clear, maybe: and deploy Antrea agent to...

Didnt get you?

Reply:
I meant this sentence is not very clear to me:
> It will deploy a single replica of Antrea controller and Antrea agent to every worker nodes of the GKE cluster.
Might be better:
> It will deploy a single replica of Antrea controller to the GKE cluster and deploy Antrea agent to every Node.

Reply:
Done